### PR TITLE
Adding GetRequestingPartyToken method

### PR DESCRIPTION
**Pull request by agorman**
I need to be able to get a RPT. This could potentially be rolled into `GetToken` and `TokenOptions` but seemed a bit messy so I made it a different method.

### CODE REVIEW

File: client_test.go [hunk 1]
```diff
@@ -604,6 +604,39 @@ func TestGocloak_GetToken(t *testing.T) {
...
+	newToken, err := client.GetToken(
+		cfg.GoCloak.Realm,
+		TokenOptions{
+			ClientID:      &cfg.GoCloak.ClientID,
+			ClientSecret:  &cfg.GoCloak.ClientSecret,
+			Username:      &cfg.GoCloak.UserName,
+			Password:      &cfg.GoCloak.Password,
+			GrantType:     StringP("password"),
+			ResponseTypes: []string{"token", "id_token"},
+			Scopes:        []string{"openid", "offline_access"},
```
Comment (SVilgelm) [inline client_test.go]: Do you really need an offline token for GetRequestingPartyToken?

Comment (agorman) [reply client_test.go]: no, I just copied the `TokenOptions` from the `GetToken` test. I can remove that if you'd like.

Comment (SVilgelm) [reply client_test.go]: yes, remove please

**Comment by SVilgelm** [comment]
Please fix goblin/govet issues

**Comment by SVilgelm** [comment]
Something incorrect in the changes for realm.json. I run the unit tests against master and everything works fine: https://travis-ci.com/Nerzal/gocloak/builds/141319818

**Comment by SVilgelm** [comment]
Same error:
```
17:25:07,047 WARN  [org.keycloak.services] (ServerService Thread Pool -- 72) KC-SERVICES0005: Unable to import realm gocloak from file /tmp/gocloak-realm.json.: java.lang.RuntimeException: Script upload is disabled
399
```

**Comment by agorman** [comment]
Oh I see. This is because the default policy created by Keycloak when enabling authorization is a JS script and uploading scripts is disabled by default after Keycloak 7.0.1.

I can either try to make some other catch all policy so the test passes or Keycloak needs to be run with `-Dkeycloak.profile.feature.upload_scripts=enabled`.

**Comment by SVilgelm** [comment]
if it possible to add the `-Dkeycloak.profile.feature.upload_scripts=enabled` parameter to travis script, then do it

**Comment by SVilgelm** [comment]
Released in v4.2.0: https://github.com/Nerzal/gocloak/releases/tag/v4.2.0

**Pull request merged**